lib/components/RadioButton/RadioButton.styled.ts

```diff
@@ -1,57 +1,64 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-export const RadioButtonInput = styled.input`
-  position: absolute;
-  width: ${({ theme }) => theme.spacings.md};
-  height: ${({ theme }) => theme.spacings.md};
-  margin: 0;
-  padding: 0;
+export const RadioButtonInput = styled.input<{ checked?: boolean }>`
+  display: inline-block;
+  appearance: none;
 
-  opacity: 0;
-`;
-
-export const RadioButtonRadio = styled.div<{ checked?: boolean }>`
   position: relative;
-  z-index: -1;
+  top: 3px;
```

Comment:
this is causing misalignment between the button and the text.

Reply:
Huum, I actually added that to AVOID the misalignment. It shows aligned to me (WITH the top: 3px)... that's weird. I will check